rules/eslint/best-practices/on.js

```diff
@@ -140,7 +140,7 @@ module.exports = {
     // requires to declare all vars on top of their containing scope
     "vars-on-top": "off",
     // require immediate function invocation to be wrapped in parentheses
-    "wrap-iife": "off",
+    "wrap-iife": ["error", "inside"],
```

Comment:
This one may be a bit contentious but in truth, now that everything runs through babel, I don't see as many iffes as I used to in code so maybe this is nbd at all